ow_power_system/src/power_system_node.cpp

```diff
@@ -24,6 +24,10 @@ const string FAULT_NAME_THERMAL = "thermal_power_failure";
 //
 static constexpr int TEMPERATURE_INDEX = 1;
 
+// Vars used to handle preventing more than one fault activation.
+static bool OneFaultInjected = false;
+static string SelectedFaultName = "";
```

Comment:
After noticing that these variables are only used in one function, an improvement would be moving these definitions into that function (`injectFault`) as static _function_ variables.  This keeps the definitions as local as possible.   As function variables, their names should go back to lowercase/underscore.